nextest-runner/src/reuse_build/archiver.rs

```diff
@@ -217,9 +217,7 @@ impl<'a, W: Write> Archiver<'a, W> {
                 encoder
                     .include_checksum(true)
                     .map_err(ArchiveCreateError::OutputArchiveIo)?;
-                encoder
-                    .multithread(get_num_cpus() as u32)
-                    .map_err(ArchiveCreateError::OutputArchiveIo)?;
+                let _err_ignored = encoder.multithread(get_num_cpus() as u32);
```

Comment:
Thank you for debugging this and for the PR!

Could you add a warning about this rather than ignoring the error? Just using `tracing::warn!` would be fine. (If you don't like `warn!` then `info!` is ok.)

Reply:
Sure! I wasn't to sure which methodology is best for giving a warning. I'll add it asap.

Reply:
Updated, and rebased with `main` as well.

Reply:
@sunshowers: Updated and rebased with `main`, as well.